index.html

```diff
@@ -177,33 +263,34 @@ <h5 class="connect">LET'S CONNECT</h5>
                 <li><img src="./img/Ellipse.png" alt="CSS"></li>
                 <li><p>CSS</p></li>
                   </ul>
+            </div>
             <ul class="languages">
                 <li><p>Frameworks</p></li>
-                <li><img src="./img/arrow1.png" alt="arrow"></li>
+                <li><img src="./img/arrow2.png" alt="arrow2"></li>
                 </ul>
             <ul class="languages">
                 <li><p>Skills</p></li>
-                <li><img src="./img/arrow1.png" alt="arrow"></li>
+                <li><img src="./img/arrow2.png" alt="arrow2"></li>
                 </ul>
        </div>
-       </div>
      </div>
-
-
-    <section>
-      <div class="form-container">
+<!-- End about myself...................................... -->
+<!-- Start Contact Form ........... -->
+      <section class="form-container">
         <div class="form">
-          <form action="https://formspree.io/f/xgeqvalw" method="post">
+            <form action="https://formspree.io/f/xgeqvalw" method="post">
+            <div class="contenido-form">
             <h2 class="title1"> Contact me </h2>
             <p class="form-p"> If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
             </p>
             <input type="text" name="fullname" class="form-control" placeholder="Yeremias" maxlength="30" required>
             <input type="email" name="emailaddress" class="form-control" placeholder="natajayanj14@gmail.com" required>
             <textarea  class="form-control" name="txt_area" placeholder="Write your message here" maxlength="500" required></textarea>
             <button type="submit" class="form_btn"> get it touch </button>
```

Comment:
- [ ] Please create the style for the button pressed following the Figma template. It's a project requirement.